.stylelintrc.json

```diff
@@ -50,6 +50,6 @@
         "severity": "warning"
       }
     ],
-    "import-notation": "less"
+    "import-notation": "string"
```

Comment:
Whilst this was not causing failures in the linting, less was apparently not a valid value and was causing the npm test step to fail